src/zenml/analytics/context.py

```diff
@@ -268,6 +270,16 @@ def track(
             }
         )
 
+        # Timezone as tzdata
+        tz = datetime.datetime.now(datetime.timezone.utc).astimezone().tzname()
+        if tz is not None:
+            properties.update({"timezone": tz})
+
+        # Language code such as "en_DE"
+        language_code = locale.getdefaultlocale()[0]
```

Comment:
From the docs:

> "Deprecated since version 3.11, will be removed in version 3.15."

https://docs.python.org/3/library/locale.html#locale.getdefaultlocale

Also this feels a bit brittle with the `[0]` indexing. Could you make it a bit more robust?

Reply:
Good catch, just changed it!